monorepo/scripts/utils.js

```diff
@@ -0,0 +1,132 @@
+// @ts-check
+const fs = require('fs');
+const path = require('path');
+const spawn = require('child_process').spawn;
+
+const renameFolder = (oldPath, newPath) => {
```

Comment:
why? i we have sparky. I had 5 lines of config man ;-) you replaced it with 5000 ;-)

Reply:
removed, was me getting carried away :-)